packages/builder/src/components/start/AppRow.svelte

```diff
@@ -2,14 +2,19 @@
   import FavouriteAppButton from "@/pages/builder/portal/workspaces/_components/FavouriteAppButton.svelte"
   import { contextMenuStore } from "@/stores/builder"
   import { auth } from "@/stores/portal"
+  import { goto as gotoStore } from "@roxi/routify"
   import { Body, Button, Icon } from "@budibase/bbui"
   import { UserAvatars } from "@budibase/frontend-core"
   import { sdk } from "@budibase/shared-core"
   import { processStringSync } from "@budibase/string-templates"
-  import { goto } from "@roxi/routify"
   import AppContextMenuModals from "./AppContextMenuModals.svelte"
   import getAppContextMenuItems from "./getAppContextMenuItems.js"
 
+  // Initialize Routify store and derive callable function
+  $gotoStore
+  let goto
+  $: goto = $gotoStore
```

Comment:
NAB: Is this necessary? Can you not just reference `$gotoStore` instead of `goto` ?